src/packages/graphweaver-helpers/src/provider/index.ts

```diff
@@ -82,7 +82,6 @@ export const createProvider = <Entity extends WithId, Context, DataEntity extend
 		}
 
 		_mapDataEntity(dataEntity: DataEntity): DataEntity {
-			console.log('_mapDataEntity ', this?.dataEntity);
```

Comment:
ty I think I left this here.

Reply:
This line has been removed after merging with main.